pkg/sql/backfill.go

```diff
@@ -58,7 +58,7 @@ const (
 
 	// checkpointInterval is the interval after which a checkpoint of the
 	// schema change is posted.
-	checkpointInterval = 10 * time.Second
+	checkpointInterval = 2 * time.Minute
```

Comment:
I wish we could set this to schemachanger.lease.duration * schemachanger.lease.renew_fraction * 0.8 but I'm sure that will break some unittest.

Reply:
Did you want me to try that? or just merge this one for now?